pkgs/by-name/hy/hyprnotify/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  pkgs,
```

Comment:
`pkgs` is not allowed here. `callPackage` will expand all attributes in `pkgs` and pass the arguments needed by this function.

For example, if you need `buildGoModule`, you just write `buildGoModule` here. (side note: It's a good idea to match the order in which these arguments are used. However, we usually put `lib` first.)

Reply:
i think my recent commit fixes this, please LMK if there is anything else!